lib/src/runner/configuration/load.dart

```diff
@@ -371,7 +371,7 @@ class _ConfigurationLoader {
           (valueList) =>
               valueList.every((value) => _packageName.hasMatch(value)));
 
-      return valueNode.value;
+      return new List<String>.from(valueNode.value);
```

Comment:
[optional] could also be `valueNode.value.cast<String>();`

Reply:
Then you'd have to cast `value` to a List, since it's dynamic...trying to keep it simple